pkg/scheduler/core/division_algorithm.go

```diff
@@ -140,13 +168,6 @@ func divideReplicasByAvailableReplica(clusterAvailableReplicas []workv1alpha2.Ta
 	}
 	divideRemainingReplicas(int(replicas-allocatedReplicas), desireReplicaInfos, clusterNames)
 
-	// For scaling up
-	for _, cluster := range unusedClusters {
-		if _, exist := desireReplicaInfos[cluster]; !exist {
-			desireReplicaInfos[cluster] = 0
```

Comment:
That's right, I made a deletion here. @qianjun1993

Reply:
my mistake